core-bundle/src/Pow/Altcha/Altcha.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Pow\Altcha;
+
+use Contao\CoreBundle\Entity\PowAltcha;
+use Contao\CoreBundle\Pow\Altcha\Config\AlgorithmConfig;
+use Contao\CoreBundle\Pow\Altcha\Exception\InvalidAlgorithmException;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\EntityManagerInterface;
+
+class Altcha
```

Comment:
```suggestion
/**
* @internal 
*/
class Altcha
```

This will allow us to change the signature.